Calculator.yashsachdev/Program.cs

```diff
@@ -0,0 +1,146 @@
+﻿// See https://aka.ms/new-console-template for more information
+global using System;
+using Calculator;
+
+bool endApp = false;
+
+Console.WriteLine("Console Calculator in c#\r");
+Console.WriteLine("------------------------\n");
+
+var calculator = new Calculator.yashsachdev.Calculator(); // creating a instance of class
+
+while (!endApp)
+{
+    string numInput1 = "";
+    string numInput2 = "";
+    double result = 0;
+
+    Console.Write("Type a number, and then press Enter: ");
+    numInput1 = Console.ReadLine();
+
+    double cleanNum1 = 0;
+    while (!double.TryParse(numInput1, out cleanNum1))
+    {
+        Console.Write("This is not valid input. Please enter an integer value: ");
+        numInput1 = Console.ReadLine();
+    }
+
+    if (calculator.cnt == 0) // Condition so user enter 2nd number and not list.
+    {
+        Console.WriteLine("Type Another Number");
+    }
+
+    else
+    {
+        Console.Write("Type another number OR type 'list' to use results from previous calculations , and then press Enter: ");
+    }
+
+    numInput2 = Console.ReadLine();
+    if (numInput2 == "list" && calculator.store.Count != 0) // Challenge 3: Allow the users to use the results in the list above to perform new calculations. 
+    {
+        calculator.DisplayResult(); //function to display list and its indexes.
```

Comment:
It's ok to comment on code for learning, or when in development, but in general refrain from commenting, unless there's something here not explained by the code itself. If classes members (fields, methods, properties) are well-named, almost no comments are necessary.